### PR TITLE
Tech-preview-release-notes

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: documentation/docs/release-notes/tech-preview.md
Comment (capri1989) [inline documentation/docs/release-notes/tech-preview.md:1]: Given that this is the first "release", I would like to have a full-blown feature document and not only enhancements and new features.

Comment (capri1989) [reply documentation/docs/release-notes/tech-preview.md:1]: or do you think this should be put into a blog and part of the final GA release?

File: documentation/docs/release-notes/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* Fixed a bug where `pg_tde` sent many 404 requests to the Vault server
+* Fixed сompatibility issues with old OpenSSL versions
+* Fixed сompatibility with old Curl versions 
+
+## MVP (2023-12-12)
+
+The Mininum Vial Product (MVP) version introduces the following functionality:
+
+* Encryption of heap tables, including TOAST
+* Encryption key are stored either in Hashicorp Vault server or in local keyring file (for development)
```
Comment (capri1989) [inline documentation/docs/release-notes/release-notes.md:43]: "Encryption keys" instead of key

**Review by capri1989 — APPROVED** [review]
LGTM

**Comment by codeforall** [comment]
We decided to use Alpha1 as a release version instead of TechPreview. We need to reflect this in the release notes update.

File: documentation/docs/release-notes/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# pg_tde release notes
+
+ ## Technical preview (2024-03-28)
```
Comment (capri1989) [inline documentation/docs/release-notes/release-notes.md:3]: Should this here also state Alpha1 now?

**Review by codeforall — APPROVED** [review]

**Review by dutow — APPROVED** [review]

**Pull request merged**